api/mapping/mapmodule/request/rearrangeselectedmaplayerrequest.md

```diff
@@ -0,0 +1,37 @@
+# RearrangeSelectedMapLayerRequest [rpc]
+
+Allow user to change selected map layer position (upper/lower) on a map.
+
+## Use cases
+
+- change selected map layer position
+
+## Description
+
+Requests a map layer position to be changed on a map.
```

Comment:
```suggestion
Requests order of a map layer to be changed in relation to other layers on the map.
```

Reply:
Should document what happens if the index/position is out of bounds. But it's already good that we have atleast some documentation for this.